alarm.graphqls

```diff
@@ -37,6 +38,7 @@ input AlarmTag {
     value: String
 }
 
+
```

Comment:
```suggestion
```

Be careful, don't change an unnecessary thing, due to not confusing others.

Reply:
> Be careful, don't change an unnecessary thing, due to not confusing others.

Yeah, I'm so sorry. I'll keep it in mind.

Reply:
Yeah, Thanks a lot.